src/components/felles/dobbelinfo.tsx

```diff
@@ -0,0 +1,24 @@
+import '../info.css';
+import { StringOrNothing } from '../../utils/felles-typer';
+
+interface Props {
+    header: string;
+    values?: StringOrNothing[];
+    children?: React.ReactNode;
+}
+export function DobbeltInformasjon(props: Props) {
+    return (
+        <div className="info_span">
+            <span className="tittel">{props.header}:</span>
+            <span className='infoboks_container'>
+                {props.values?.map((value, index) => (
+                    <span key={index} className="undertittel">
+                        {value}
+                    </span>
+                ))}</span>
+            {props.children}
+        </div>
```

Comment:
Jeg syns fortsatt dere bør prøve å bruke [typografi-komponentene fra designsystemet](https://aksel.nav.no/komponenter/core/typography), med mindre dere syns det blir altfor mye justeringer som må til for å få dem til å passe. Har ikke fått med meg hva dere har tenkt 😬